src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -71,9 +71,14 @@
     short-summary: Commands to manage a DC/OS-orchestrated Azure Container Service.
 """
 
-helps['acs install-cli'] = """
+helps['acs dcos install-cli'] = """
```

Comment:
Nice--thank you for catching and fixing this.

Reply:
Soon I plan to add a script that will help validate the help files against the command table (which essentially is how I caught this).